jest-db.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    testTimeout: 5000000,
+    setupFilesAfterEnv: ['<rootDir>/setupTests.js', '<rootDir>/node_modules/jest-enzyme/lib/index.js'],
```

Comment:
since we are doing this for the first time in this repo I want to change the name setupTests.js to jest-test-setup.js - first: mixed case file names are a problem on some file systems (like windows), and second because the name is too general - adding jest to it makes is more clear.